app/src/test/java/org/mozilla/focus/utils/ToastManagerTest.kt

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils
+
+import android.content.Context
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.focus.R
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.shadows.ShadowToast
+import java.util.Locale
+
+@RunWith(RobolectricTestRunner::class)
+class ToastManagerTest {
+
+    private lateinit var context: Context
+
+    private val lastToastText: String get() = ShadowToast.getTextOfLatestToast()
+    private val lastToastTextLowercase: String get() = lastToastText.toLowerCase(Locale.ENGLISH)
+
+    @Before
+    fun setUp() {
+        context = RuntimeEnvironment.application
+    }
+
+    @Test
+    fun `WHEN showUnpinnedToast is called THEN it did not leave a string substitution template around`() {
```

Comment:
Could then THEN section of this method name be clarified a bit?

Reply:
```
fun `GIVEN showUnpinnedToast creates the toast text with a string substitution WHEN showUnpinnedToast is called THEN the toast text does not contain the string substitution template literal`() {
```